web/modules/custom/mymodule/src/Controller/FirstController.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @file 
+ * Generate markup to be displayed. Functionality in this controller is
+ * wired to Drupal in mymodule.routing.yml.
+ */
+
+namespace Drupal\mymodule\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Cache;
+/**
+ * This class is a multipurpose utility class.
+ * it is used for testing simple markups,getAccountName function.  
+ */
+class FirstController extends ControllerBase {
+
+  /**
+   * @return [array]
```

Comment:
Please check in Drupal core how the function docblock is created.
This way is incorrect.